tests/src/lib.rs

```diff
@@ -41,26 +41,28 @@ pub fn get_test_file_path(path: String) -> Result<String, String> {
             Ok(pb) => Ok(String::from(pb.to_str().unwrap())),
             Err(e) => Err(e),
         }
-
     } else {
         panic!("Cannot find the absolute path to {}. Check that this function being called from test code, not production code.", this_source_file);
     }
 }
 
-fn next_test_file_path(maybe_absolute_dir: &std::path::Path, file_path: &String) -> Result<PathBuf, String> {
+fn next_test_file_path(
+    maybe_absolute_dir: &std::path::Path,
+    file_path: &String,
+) -> Result<PathBuf> {
     let path_to_test = maybe_absolute_dir.join(file_path);
     if path_to_test.exists() {
         Ok(path_to_test)
     } else if maybe_absolute_dir.parent().is_some() {
         next_test_file_path(maybe_absolute_dir.parent().unwrap(), file_path)
     } else {
-        Err(format!("Cannot locate file '{}'", file_path))
+        Err(anyhow!("Cannot locate file '{}'", file_path))
```

Comment:
Ah! You can also just do `bail!("Cannot locate file '{file_path}'")` as a shorthand